ERCS/erc-draft_composite_policy.md

```diff
@@ -0,0 +1,425 @@
+---
```

Comment:
```suggestion
---
eip: 8006
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.